### PR TITLE
fix(red): HA

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: managed-databases/redis/concepts.mdx [hunk 1]
```diff
@@ -35,7 +35,9 @@ A Database Instance is made up of multiple (at least 1) dedicated compute nodes,
 
 ## High availability
 
-High Availability is handled through the Redis™ clustering feature. This is a 3-node multi-master architecture. In case of downtime on a node in the cluster, a new node is able to take over the requests with minimal downtime.
+Is a Database Instance configuration that allows you to create a standby node, with an up-to-date replica of the database. If the main node fails for any reason, the standby can take over requests, reducing downtime.
```
Comment (nerda-codes) [inline managed-databases/redis/concepts.mdx:38]:
```suggestion
High Availability is a Database Instance configuration that allows you to create a standby node, with an up-to-date replica of the database. If the main node fails for any reason, the standby can take over requests, reducing downtime.
```

File: managed-databases/mongodb/how-to/delete-a-database-instance.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    </Message>
+4. Type **DELETE** to confirm and click **Delete this Instance**.
+    <Message type="tip">
+      Alternatively, you can delete your Instance from the Database Instances list by clicking <Icon name="more" /> next to the Instance name and then **Delete**.
+    </Message>
+
+
+<Navigation title="See also">
+  <PreviousButton to="/managed-databases/document-database/how-to/delete-database">How to delete a database</PreviousButton>
+</Navigation>
```
Comment (nerda-codes) [inline managed-databases/mongodb/how-to/delete-a-database-instance.mdx:37]:
```suggestion
```

Comment (nerda-codes) [reply managed-databases/mongodb/how-to/delete-a-database-instance.mdx:37]: Previous and next pages are now automatically handled (defined by the order in the nav menu)

File: managed-databases/mongodb/how-to/delete-a-database-instance.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+categories:
+  - managed-databases
+---
+
+<Macro id="requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [MongoDB™ Database Instance](/managed-databases/document-database/quickstart)
+</Message>
```
Comment (nerda-codes) [inline managed-databases/mongodb/how-to/delete-a-database-instance.mdx:21]:
```suggestion
  - A Scaleway account logged into the [console](https://console.scaleway.com)
  - Created a [MongoDB™ Database Instance](/managed-databases/document-database/quickstart)
```

File: faq/databases-for-redis.mdx [hunk 4]
```diff
@@ -90,15 +90,15 @@ Refer to the image below to see which node types belong to which node class.
...
 This is much better for scaling as the operation is spread across multiple nodes instead of having a single entry point.
 
 <Lightbox src="scaleway-redis-cluster.webp" alt="" />
 
-A two-node architecture for Redis™ with a primary node being duplicated in a standby node might increase the durability of the data but does not prevent the service from failing as automatic failover is not available for Redis databases.
-This means that the primary node will be a Single Point Of failure.
-Scaleway does not provide two nodes HA Redis™ Database Instances but offers a state-of-the-art Redis Cluster Architecture, which can scale vertically and horizontally with built-in High Availability.
+Two-node High Availability configurations are available with Redis™ Database Instances. This configuration type allows you to create a standby node, with an up-to-date replica of the database. If the main node fails for any reason, the standby can take over requests, reducing downtime.
+
+The HA standby node is linked to the main node, using asynchronous replication. Asynchronous replication allows to keep good performances.
```
Comment (RoRoJ) [inline faq/databases-for-redis.mdx:101]:
```suggestion
The HA standby node is linked to the main node, using asynchronous replication. Asynchronous replication allows you to maintain good performance.
```

File: managed-databases/redis/concepts.mdx [hunk 5]
```diff
@@ -35,7 +35,9 @@ A Database Instance is made up of multiple (at least 1) dedicated compute nodes,
 
 ## High availability
 
-High Availability is handled through the Redis™ clustering feature. This is a 3-node multi-master architecture. In case of downtime on a node in the cluster, a new node is able to take over the requests with minimal downtime.
+High Availability is a Database Instance configuration that allows you to create a standby node, with an up-to-date replica of the database. If the main node fails for any reason, the standby can take over requests, reducing downtime.
+
+The HA standby node is linked to the main node, using asynchronous replication. Asynchronous replication allows to keep good performances.
```
Comment (RoRoJ) [inline managed-databases/redis/concepts.mdx:40]:
```suggestion
The HA standby node is linked to the main node, using asynchronous replication. Asynchronous replication allows you to maintain good performance.
```

File: managed-databases/redis/reference-content/ensuring-data-persistence.mdx [hunk 6]
```diff
@@ -49,8 +49,6 @@ A Redis cluster contains a minimum of three nodes and up to six nodes. Each node
...
 ## High Availability
 
-The Scaleway Managed Databases team is currently working on a High Availability configuration that will allow standalone nodes to be deployed with a replica node.
+High Availability is a Database Instance configuration that allows you to create a standby node, with an up-to-date replica of the database. If the main node fails for any reason, the standby can take over requests, reducing downtime.
 
-This feature is expected in October 2023.
-
-Follow our [changelog](/changelog/) to remain up-to-date with the latest improvements in Managed Databases for Redis™.
\ No newline at end of file
+The HA standby node is linked to the main node, using asynchronous replication. Asynchronous replication allows to keep good performances.
```
Comment (RoRoJ) [inline managed-databases/redis/reference-content/ensuring-data-persistence.mdx:54]:
```suggestion
The HA standby node is linked to the main node, using asynchronous replication. Asynchronous replication allows you to maintain good performance.
```

**Review by RoRoJ — APPROVED** [review]

**Review by scw-wtimmermans — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**